7.4/Dockerfile-cli

```diff
@@ -0,0 +1,25 @@
+FROM php:7.4
```

Comment:
Please use `php:7.4-alpine`

Reply:
Right, tested it on rc, and removed too much of the string for stable :)